packages/git/src/browser/git-widget.tsx

```diff
@@ -821,9 +900,10 @@ export class GitChangesListContainer extends React.Component<GitChangesListConta
     protected renderUnstagedChanges(repository: Repository | undefined): React.ReactNode | undefined {
         if (this.props.unstagedChanges.length > 0) {
             return <div id='unstagedChanges' className='changesContainer'>
-                <div className='theia-header'>
+                <div className='theia-header git-theia-header'>
                     Changes
                     {this.renderChangeCount(this.props.unstagedChanges.length)}
+                    {this.renderChangeListButtons()}
```

Comment:
You should only render the 'stage all' button and in addition the 'discard all' button here.